eclass/tree-sitter-grammar.eclass

```diff
@@ -42,13 +42,7 @@ EXPORT_FUNCTIONS src_compile src_install
 # This internal function determines the ABI version of a grammar library based
 # on the package version.
 _get_tsg_abi_ver() {
-	if ver_test -gt 0.21; then
-		die "Grammar too new; unknown ABI version"
-	elif ver_test -ge 0.19.0; then
-		echo 13
-	else
-		die "Grammar too old; unknown ABI version"
-	fi
+	sed -r -n '/#define LANGUAGE_VERSION ([0-9]*)$/{s@@\1@;p}' "${S}"/parser.c
```

Comment:
Hmm this didn't work for me, also I think we need to die on failure.

I tried to simplify it a bit and ended up with this:

```suggestion
	sed -n '/#define LANGUAGE_VERSION/s/.* //p' "${S}"/parser.c ||
	  die 'Failed to extract ABI version'
```

Reply:
> this didn't work for me

Uhm... Strange... I tested it on multiple grammars (like `-c`, `-haskell` before commiting. But ok, I'll apply your suggested changes in few minutes.

Reply:
+1 on the die